scripts/vscode-task.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2025 Intel Corporation. All rights reserved.
+
+# Simple helper script for vscode task support.
+# Current vscode tasks have difficulty executing multiple commands.
+
+# setup environment
+source ~/zephyrproject/.venv/bin/activate
```

Comment:
same comment as for #9878 - too many hard-coded assumptions. Let's not do this

Reply:
How about make this conditional, if test -e ~/zephyrproject/.venv/bin/activate handling the default config?

I don't really mind for vscode-task.sh -- this can make the default scase easier and non-default setups can use a symlink or something like that.